QuasistaticBrownianThermalNoise/QuasistaticBrownianThermalNoise.cpp

```diff
@@ -1146,12 +1337,47 @@ void ElasticProblem<dim>::output_results(const unsigned int cycle) {
 
       // get function gradients at the quadrature points:
       // solution_gradients[q][c][d] is deriv in direction d of cth vector
-      // at quadrature point q. So u_{j;i}(x_q) = solution_gradients[q][j][i]
+      // at quadrature point q. So u_{j;i}(x_q) =
+      // solution_gradients[q][j][i]
       fe_values.get_function_gradients(locally_relevant_solution,
                                        solution_gradients);
 
       // store current quadrature point's Young's modulus
       double theYijkl = 0.;
+      double sYijkl = 0;
+
+      for (unsigned int q_point = 0; q_point < n_q_points; ++q_point) {
+        for (int i = 0; i < dim; ++i) {
+          for (int j = 0; j < dim; ++j) {
+            for (int k = 0; k < dim; ++k) {
+              for (int l = 0; l < dim; ++l) {
+                local_U_this_energy = 0.5 * fe_values.JxW(q_point) *
+                                      solution_gradients[q_point][j][i] *
+                                      solution_gradients[q_point][l][k];
+
+                // only perform the energy calculation for points above 0
+                // since we are only interested in the coating.
+                if (fe_values.quadrature_point(q_point)(dim - 1) > 0.) {
+                  // If custom loss angles are specified, then we calculate
+                  // the energy for the coating using the appropriate youngs
+                  // tensor.
+                  if (config_multloss["CustomLossAngles"].as<bool>()) {
```

Comment:
It sounds like this parameter is turning MultipleLossAngles on or off, not letting you specify in the config file your own loss angles. So rename this parameter to "UseMultipleLossAngles"?